build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json

```diff
@@ -2,7 +2,7 @@
   "config_name": "cpu_llvm_sync",
   "iree_compile_flags": [
     "--iree-hal-target-backends=llvm-cpu",
-    "--iree-input-demote-f64-to-f32"
+    "--iree-input-demote-f64-to-f32=false"
```

Comment:
false is the default right? You can just leave the flag off then.

Reply:
It is not. I have to set to false to disable.

Reply:
https://github.com/iree-org/iree/blob/6c45befbf9efa47e52471e612fedded32f05f0ee/compiler/src/iree/compiler/Pipelines/Options.h#L53-L58 ah. Weird that we have it explicitly set to false in so many places then.